lab_5_scrapper/scrapper.py

```diff
@@ -120,45 +222,83 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self._config = config
+        self.article = Article(full_url, article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        content_bs = article_soup.find('div', {'class': 'news-detail__text'})
+        if content_bs:
+            text = ''
+            for p in content_bs.find_all('p'):
+                text += p.text.strip()
+            self.article.text = text
 
     def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
         """
         Finds meta information of article
         """
-        pass
+        title = article_soup.find('h1')
+        if title:
+            self.article.title = title.text.strip()
 
-    def unify_date_format(self, date_str: str) -> datetime.datetime:
+        span_author = article_soup.find('span', itemprop='name')
+        if span_author:
```

Comment:
task description contains information concerning what to do if there is not author

Reply:
done!